docs/concepts/09_glossary.md

```diff
@@ -0,0 +1,134 @@
+
+### Verifier
+
+A person, organization, or system that checks the validity and authenticity of an individual's credentials or qualifications. Verifiers play a critical role in building trust and ensuring that credentials are accurate and reliable.
+
+## Eco-system terms
+
+### Blockchain Technology
+
+A type of distributed ledger technology that allows multiple parties to have a synchronized, transparent, and immutable record of transactions. It uses cryptographic techniques to secure and verify transactions, and it does not require a central authority or intermediaries.
+
+### Decentralized Network
+
+A network of computers or nodes that operate without a central authority or control. In a decentralized network, each node has equal control over the network, and decisions are made through a consensus mechanism, rather than by a single entity or group.
+
+### Distributed Ledger Technology (DLT)
+
+A type of digital database that stores information across a network of computers or nodes. It allows multiple parties to have a synchronized, transparent, and immutable record of transactions, without the need for a central authority or intermediaries.
+
+### Extrinsics
+
+A transaction that originates from an external account and affects the state of the blockchain. It can be used to execute actions on the network, such as transferring funds, making governance decisions, use functionality of the parachain or interacting with smart contracts. More details about Extrinsics can be found at the [offical documentation](https://wiki.polkadot.network/docs/learn-extrinsics)
+
+### Parachain
+
+A specialized blockchain that runs on top of the Polkadot network and is connected to the main network through a relay chain. More details about Parachains can be found at the [offical documentation](https://wiki.polkadot.network/docs/learn-parachains)
+
+### Polkadot
+
+A multi-chain network that allows for interoperability between different blockchain protocols, including the KILT Protocol. More details about Polkadot can be found on their [offical documentation](https://wiki.polkadot.network/docs/getting-started#what-is-polkadot)
+
+### Substrate
+
+A modular blockchain development framework used to build custom blockchain solutions, including the KILT Protocol. More details about Substrate can be found on their [offical documentation](https://docs.substrate.io/)
+
+### Relay Chain
+
+The central chain in the Polkadot network that coordinates communication and consensus between different parachains. More details about Parachains can be found at the [offical documentation](https://wiki.polkadot.network/docs/learn-architecture)
+
+I hope this glossary helps clarify some of the key terms related to the KILT Protocol!
```

Comment:
Who's I?

```suggestion
I hope this glossary helps clarify some of the key terms related to the KILT Protocol!
```

Reply:
```suggestion
KILT Team hope this glossary helps clarify some key terms related to the KILT Protocol!
```

Reply:
I would remove the sentence. Looks like it belongs to the `Relay Chain` section.